### PR TITLE
doc: add note for use install.sh

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
Signed-off-by: xiang <xiang13225080@163.com>

### CODE REVIEW

File: docs/user_guides/installation.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ If you are using k3s or k3d, please also specify `--k3s` flag.
   curl -sSL https://mirrors.chaos-mesh.org/latest/install.sh | bash -s -- --k3s
 </PickVersion>
 
+**Note:**
+>
+> `install.sh` is suitable for trying out the chaos Mesh. Helm is recommended for installation if Chaos Mesh is used in a production environment or if you want to use it deeply.
```
Comment (dcalvin) [inline docs/user_guides/installation.md:39]:
```suggestion
> `install.sh` is suitable for trying Chaos Mesh out. If you want to use Chaos Mesh in production or other serious scenarios, Helm is the recommended deployment method.

Comment (dcalvin) [reply docs/user_guides/installation.md:39]: No "the" before Chaos Mesh, which is a proper noun.

File: docs/user_guides/installation.md [hunk 2]
```diff
@@ -34,6 +34,10 @@ If you are using k3s or k3d, please also specify `--k3s` flag.
...
+
 ### Verify your installation
 
 Verify if the chaos mesh is running (For the use of _kubectl_, you can refer to the [documentation](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands).)
```
Comment (dcalvin) [inline docs/user_guides/installation.md:43]:
```suggestion
Verify if Chaos Mesh is running (For the use of _kubectl_, you can refer to the [documentation](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands).)
```

**Review by dcalvin — CHANGES_REQUESTED** [review]
Please update in all related versions.

**Comment by WangXiangUSTC** [comment]
@dcalvin all comment addressed, PTAL again

**Review by dcalvin — APPROVED** [review]
LGTM

**Pull request merged**